<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>888</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="titleFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="titleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>801</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monotype Corsiva</family>
       <pointsize>28</pointsize>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>LED Sensor Utility (ASC)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="logoFrame">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>261</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="logoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>240</width>
       <height>65</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string>Logo</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="mainTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>1101</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="testTab">
     <attribute name="title">
      <string>Test</string>
     </attribute>
     <widget class="QGroupBox" name="messageGroupBox2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>540</y>
        <width>531</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="title">
       <string> Sensor Message </string>
      </property>
      <widget class="QTextEdit" name="messageTextEdit2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>511</width>
         <height>151</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;background-color:white;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="sensorGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>531</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="title">
       <string> Sensor  </string>
      </property>
      <widget class="QLabel" name="sensorLabel1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:blue;</string>
       </property>
       <property name="text">
        <string>COM Port</string>
       </property>
      </widget>
      <widget class="QComboBox" name="sensorComboBox1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;background-color:white;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sensorButton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>75</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QLabel" name="sensorLabel2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>55</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:blue;</string>
       </property>
       <property name="text">
        <string>Sensor Type</string>
       </property>
      </widget>
      <widget class="QComboBox" name="sensorComboBox2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;background-color:white;</string>
       </property>
      </widget>
      <widget class="QLabel" name="sensorLabel3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>55</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:blue;</string>
       </property>
       <property name="text">
        <string>Slave Address</string>
       </property>
      </widget>
      <widget class="QComboBox" name="sensorComboBox3">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>51</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;background-color:white;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="messageGroupBox1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>531</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="title">
       <string> Message </string>
      </property>
      <widget class="QTextEdit" name="messageTextEdit1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>511</width>
         <height>151</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;background-color:white;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="controlGroupBox">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>440</y>
        <width>521</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="title">
       <string> Control </string>
      </property>
      <widget class="QPushButton" name="readRGBButton">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>Read Once</string>
       </property>
      </widget>
      <widget class="QLabel" name="controlLabel1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:blue;</string>
       </property>
       <property name="text">
        <string>Sensor No</string>
       </property>
      </widget>
      <widget class="QComboBox" name="controlComboBox1">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;background-color:white;</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="continuousCheckBox">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>25</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:blue;</string>
       </property>
       <property name="text">
        <string>Continuous Read</string>
       </property>
      </widget>
      <widget class="QLabel" name="controlLabel2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>55</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:blue;</string>
       </property>
       <property name="text">
        <string>Board Delay</string>
       </property>
      </widget>
      <widget class="QComboBox" name="controlComboBox2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;background-color:white;</string>
       </property>
      </widget>
      <widget class="QLabel" name="controlLabel3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>55</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:blue;</string>
       </property>
       <property name="text">
        <string>Interval</string>
       </property>
      </widget>
      <widget class="QComboBox" name="controlComboBox3">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;background-color:white;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="messageGroupBox3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1071</width>
        <height>421</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="title">
       <string> Sensor Information </string>
      </property>
      <widget class="QTableView" name="sensorTableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>1051</width>
         <height>391</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;background-color:white;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="aboutTab">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QTextEdit" name="testarchPanel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>641</width>
        <height>711</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="releasenoteTextEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>621</width>
        <height>691</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="aboutGroupBox">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>10</y>
        <width>411</width>
        <height>711</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0,0);</string>
      </property>
      <property name="title">
       <string> About </string>
      </property>
      <widget class="QLabel" name="aboutLabel1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>381</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>LED Sensor Utility (ASC)  Version 1.0.0.0</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="aboutLabel2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>55</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Accton Technology Corporation.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="aboutLabel3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>All Rights Reserved.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="authorLabel1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>620</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Author: 朱建杭(Max Chu)</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="authorLabel2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>650</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>TEL: +886-3-5770270#2422</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="authorLabel3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>680</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>E-Mail: max_chu@accton.com</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
